src/ucs/sys/stubs.c

```diff
@@ -96,3 +96,8 @@ int ucs_empty_function_do_assert()
     ucs_assert_always(0);
     return 0;
 }
+
+void ucs_empty_function_do_assert_void()
```

Comment:
btw, we can remove return value from `ucs_empty_function_do_assert()`, because it is not needed, since `ucs_assert_always()` is fatal

Reply:
agree

Reply:
in this case we will have to use datatype casting in all constructions where routine is used - bad code style

Reply:
anyway we do casting
`(uct_completion_callback_t)ucs_empty_function_do_assert_void;`

Reply:
in this case - yes, but there are more location where casting is not used:
https://github.com/openucx/ucx/blob/f8c39554ed3c865061f0e160a1f1461267f35f9d/src/ucs/config/parser.h#L344
```
#define UCS_CONFIG_TYPE_DEPRECATED {(ucs_field_type(ucs_config_parser_t, read))   ucs_empty_function_do_assert, \
                                    (ucs_field_type(ucs_config_parser_t, write))  ucs_empty_function_do_assert, \
                                    (ucs_field_type(ucs_config_parser_t, clone))  ucs_empty_function_do_assert, \
                                    (ucs_field_type(ucs_config_parser_t, release))ucs_empty_function_do_assert, \
                                    (ucs_field_type(ucs_config_parser_t, help))   ucs_empty_function_do_assert, \
```

Reply:
maybe just pass `ucs_empty_function_do_assert()` here as well?
and remove `return 0` in `ucs_empty_function_do_assert()` - since it is not needed even if `int` is a return type